Epona/include/Epona/debug/DebugImplementation.hpp

```diff
@@ -31,59 +30,45 @@ class DebugImplementation
      * during the calculation. It then proxies all the arguments to the currently
      * set callback functor.
      *
-     * @tparam VertexBuffer convex hull vertex buffer type
-     *
      * @param[in] convexHull   current convex hull instance
      * @param[in] vertexBuffer source vertices for the convex hull
      */
-    template < typename VertexBuffer >
     static void QuickhullConvexHullCall(
-        QuickhullConvexHull<VertexBuffer>& convexHull, VertexBuffer& vertexBuffer
+        epona::ConvexHull& convexHull, std::vector<glm::vec3>& vertexBuffer
     )
     {
-        GetQuickhullConvexHullCallback<VertexBuffer>()(convexHull, vertexBuffer);
+        GetQuickhullConvexHullCallback()(convexHull, vertexBuffer);
     }
 
     /**
     * @brief Initializes QuickhullConvexHull callback instance
     *
-    * @tparam VertexBuffer convex hull vertex buffer type
-    *
     * @param callback callback function object
     */
-    template < typename BufferType >
     static void SetQuickhullConvexHullCallback(
-        std::function<void(QuickhullConvexHull<BufferType>&, BufferType&)> callback
+        std::function<void(epona::ConvexHull&, std::vector<glm::vec3>&)> callback
```

Comment:
alias this `std::function` with `using` for simplicity?

Reply:
I don't really favor those anymore, I'd rather see the full expression, for clarity sake